src/AddTodoForm.jsx

```diff
@@ -0,0 +1,13 @@
+const AddTodoForm = () => {
+    return (
+        <>
+            <form action="">
+                <label id="todoTitle" title="Title"></label>
+                <input id="todoTitle"/>
+                <button>Add</button>
```

Comment:
Adding `type="submit"` to this button will let you submit the form when you click it or hit enter on the keyboard. We'll use this in the next lesson or two as well, so it wouldn't hurt to add now!